src/Network/Algorand/Node/Api.hs

```diff
@@ -62,6 +65,49 @@ data Version = Version
   deriving (Generic, Show)
 $(deriveJSON algorandSnakeOptions 'Version)
 
+newtype NanoSec = NanoSec { unNanoSec :: Integer }
+  deriving stock (Eq, Ord, Show)
+  deriving newtype (ToJSON, FromJSON)
+
+data NodeStatus = NodeStatus
```

Comment:
Hmmm, what is the order of the fields here? It would have been easier to review (in particular, make sure that there are no missing or extra fields) if they followed the order of the documentation.

Reply:
In fact, I know it is completely not obvious, but the documentation sucks, as it uses pretty non-descriptive JavaScript types. In some cases, I found it to be plain wrong. So, I think it’s safer not to rely on the documentation, and follow the reference implementation instead: https://github.com/algorand/go-algorand/blob/c0452b2d35273448ffdcca907c50998efdd7ab75/node/node.go#L59. Note that all integers there are actually unsigned 64-bit, so I propose we follow it and use `Word64` too.

(btw, you mentioned not having a `newtype` for Round and I think it would actually not be a bad idea to have such a `newtype`, similar to how the reference implementation has a typedef for it. But it will be best to put it into a separate PR.)

Reply:
I just kept the field order from George's code.
Now changed to the documentation order (which is alphabetical).

By the way, better place to look for the definitive API spec is probably
https://github.com/algorand/go-algorand/blob/88c9f49f14d04f9f209b2acbd42c5864cae9bdc6/daemon/algod/api/server/v2/handlers.go#L271
and
https://github.com/algorand/go-algorand/blob/88c9f49f14d04f9f209b2acbd42c5864cae9bdc6/daemon/algod/api/server/v2/generated/types.go#L478